interface/drugs/drugs.inc.php

```diff
@@ -44,6 +44,30 @@ function send_drug_email($subject, $body)
     }
 }
 
+/**
+ * Validates that an encounter exists for a given patient
+ *
+ * @param int $patient_id The patient ID
+ * @param int $encounter_id The encounter ID to validate
+ * @return bool|string Returns true if valid, or an error message if invalid
+ */
+function _validateEncounter($patient_id, $encounter_id)
+{
+    if (empty($encounter_id) || $encounter_id == 0) {
+        return xlt('No active encounter selected. To dispense medication, you must first select or create a valid encounter for this patient. Please return to the patient dashboard, select or create an encounter, then try again.');
+    }
+
+    $encounter = new OpenEMR\Services\EncounterService()->getOneByPidEid($patient_id, $encounter_id);
```

Comment:
minor fix is to replace with 
```
use OpenEMR\Services\EncounterService;
```
at the top of the file
and then can call it without the path
```
$encounter = (new EncounterService())->getOneByPidEid($patient_id, $encounter_id);